csi/server/plugin/opensds/controller.go

```diff
@@ -534,17 +511,17 @@ func (p *Plugin) ControllerPublishVolume(ctx context.Context,
 		canAtMultiNode = true
 	}
 
-	err = isVolumeCanBePublished(canAtMultiNode, attachReq, volSpec.MultiAttach)
+	err = c.isVolumeCanBePublished(canAtMultiNode, attachReq, volSpec.MultiAttach)
 	if err != nil {
 		return nil, err
 	}
 
-	newAttachment, errAttach := client.CreateVolumeAttachment(attachReq)
+	newAttachment, errAttach := c.Cli.CreateVolumeAttachment(attachReq)
 	if errAttach != nil {
-		msg := fmt.Sprintf("the volume %s failed to publish to node %s, error info: %v",
+		msg := fmt.Sprintf("the volume %s is failed to publish to node %s, error info: %v",
```

Comment:
s/is failed to publish/is failed to be published

Reply:
Done.